### PR TITLE
Update : added function for setting min/max scale

### PR DESCRIPTION
**Pull request by takaken1994**
- Added function for setting min/max scale.
- Fix to remove unused params.

### CODE REVIEW

File: extensions/GUI/CCScrollView/CCScrollView.cpp [hunk 1]
```diff
@@ -51,10 +51,7 @@ static float convertDistanceFromPointToInch(float pointDis)
 
 
 ScrollView::ScrollView()
-: _zoomScale(0.0f)
```
Comment (zilongshanren) [inline extensions/GUI/CCScrollView/CCScrollView.cpp:54]: I found that these deleted code are not used by any code in CCScrollView, should we also delete the declaration in the header file?  If we just leave the member variable uninitialized which I think is not a good idea.

File: extensions/GUI/CCScrollView/CCScrollView.h [hunk 2]
```diff
@@ -146,6 +146,34 @@ class CC_EX_DLL  ScrollView : public Layer, public ActionTweenDelegate
...
      */
     void setZoomScaleInDuration(float s, float dt);
+
+    /**
+     * Set min/max scale
+     *
+     * @param minScale min scale
+     * @param maxScale max scale
+     */
+    void setMinMaxScale(float minScale, float maxScale) {
```
Comment (zilongshanren) [inline extensions/GUI/CCScrollView/CCScrollView.h]: Regards to this API, it seems a little verbose if we have had the `setMinScale` and `setMaxScale` APIs.

Comment (takaken1994) [inline extensions/GUI/CCScrollView/CCScrollView.h]: I see. I will remove this function.

File: extensions/GUI/CCScrollView/CCScrollView.cpp (hunk 1 above)
Comment (takaken1994) [inline extensions/GUI/CCScrollView/CCScrollView.cpp:54]: I had forgotten to remove them from the header file. I'll do it. Thank you!

**Comment by minggo** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
Thanks @takaken1994 
@minggo  I think we could merge this PR now.

**Pull request merged**